testdata/p4_16_samples_outputs/pna-example-varIndex-1.p4-error

```diff
@@ -0,0 +1,6 @@
+hdrs.vlan_tag[0].pcp_cfi_vid[15:4];
```

Comment:
Is this coming due to debug print? Is this needed? It is not giving any information IMO and should be removed.